### PR TITLE
MWPW-161101 load sidekick button css faster if already open

### PR DESCRIPTION
**Pull request by vhargrave**
On certain pages, where it takes a while for all the content to load (such as MEP content), the sidekick's Publish and Tools buttons switch position. This is because Milo loads some css after page load, which switches the position of the publish button - https://github.com/adobecom/milo/blob/ff054af4709a30ee04b106e4bd2979432e57f750/libs/utils/sidekick-decorate.js#L32.
This PR fixes that issue. 
To test just have the sidekick open on the linked test page. 

Resolves: [MWPW-161101](https://jira.corp.adobe.com/browse/MWPW-161101)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.live/products/photoshop-lightroom
- After: https://main--cc--adobecom.hlx.live/products/photoshop-lightroom?milolibs=vhargrave-MWPW-161101-sidekick-buttons

- Before: https://main--milo--adobecom.hlx.live/
- After: https://vhargrave-MWPW-161101-sidekick-buttons--milo--adobecom.hlx.live/

### CODE REVIEW

File: libs/utils/utils.js [hunk 1]
```diff
@@ -1052,6 +1052,10 @@ async function checkForPageMods() {
 
 async function loadPostLCP(config) {
   await decoratePlaceholders(document.body.querySelector('header'), config);
+  const sk = document.querySelector('helix-sidekick');
```
Comment (mokimo) [inline libs/utils/utils.js:1055]: Should we move this into `initSidekick` and move the init sidekick method further up to be called even earlier than the LCP? 

If authors/devs/stakeholders have the sidekick open, the CWV measures are distorted anyway also in regards to say CLS.

**Review by mokimo — APPROVED** [review]

File: libs/utils/utils.js (hunk 1 above)
Comment (vhargrave) [reply libs/utils/utils.js:1055]: Tbh, I'd be down. I'll move it 🙌

File: libs/utils/utils.js [hunk 2]
```diff
@@ -1052,6 +1052,10 @@ async function checkForPageMods() {
...
+  if (sk) {
+    import('./sidekick-decorate.js').then((mod) => { mod.default(sk); });
+  }
```
Comment (robert-bogos) [inline libs/utils/utils.js:1058]: Nit: You could do 1 line in here
```suggestion
  if (sk) import('./sidekick-decorate.js').then((mod) => { mod.default(sk); });
```

**Review by robert-bogos — APPROVED** [review]

**Comment by spadmasa** [comment]
Before :
{3CF05191-D2E6-48EB-9A07-E039761F8E17}

After the fix 
 https://main--cc--adobecom.hlx.page/products/photoshop-lightroom?milolibs=vhargrave-MWPW-161101-sidekick-buttons
{5279971F-7354-4927-965F-13704ACE38FE}
https://main--cc--adobecom.hlx.page/products/photoshop?milolibs=vhargrave-MWPW-161101-sidekick-buttons
{6F9B8CCA-4EAE-41DF-A37B-C4A9121040AC}
{5EF0D677-5FCE-44C8-8AC9-4E442294386D}

**Comment by mokimo** [comment]
@vhargrave mind adding a milo based link for the PSI check and also the automation to be able to properly merge this?

**Pull request merged**